headers/types/ground_mode/enums.h

```diff
@@ -18,104 +38,394 @@ enum script_var_id {
+    // TODO: This is probably backed by MONEY_STORED? Need to confirm.
     VAR_BANK_GOLD = 61,
+    // The species ID of the initial starter. Saved at [NA]0x2048868, which is done by the special
+    // process INIT_MAIN_TEAM_AFTER_QUIZ.
+    // This remains unchanged even if Chimecho Assembly entry 0 is modified in any way.
     VAR_HERO_FIRST_KIND = 62,
+    // The name of the initial starter. Saved at [NA]0x2048878, which is done by the special process
+    // INIT_MAIN_TEAM_AFTER_QUIZ.
+    // This remains unchanged even if Chimecho Assembly entry 0 is modified in any way.
     VAR_HERO_FIRST_NAME = 63,
+    // The species ID of the initial partner. Saved at [NA]0x20488D8, which is done by the special
+    // process INIT_MAIN_TEAM_AFTER_QUIZ.
+    // This remains unchanged even if Chimecho Assembly entry 1 is modified in any way.
     VAR_PARTNER_FIRST_KIND = 64,
+    // The name of the initial partner. Saved at [NA]0x20488E8, which is done by the special process
+    // INIT_MAIN_TEAM_AFTER_QUIZ.
+    // This remains unchanged even if Chimecho Assembly entry 1 is modified in any way.
     VAR_PARTNER_FIRST_NAME = 65,
+    // The "talk kind" of the initial starter, typically used in scripts where the player speaks;
+    // see enum talk_kind (4, 5). This is most likely done to differentiate speech patterns between
+    // genders (this probably has a purpose in Japanese, just not in English).
+    // Saved at [NA]0x204893C, which is done by the special process INIT_MAIN_TEAM_AFTER_QUIZ. This
+    // remains unchanged even if Chimecho Assembly entry 0 is modified in any way.
     VAR_HERO_TALK_KIND = 66,
+    // The "talk kind" of the initial partner, typically used in scripts where the partner speaks to
+    // differentiate speech patterns; see enum talk_kind (1, 2, 3). Unlike HERO_TALK_KIND, there's
+    // actually a table that maps starters to certain talk kinds ([NA]0x209CCE4). Each entry
+    // consists of a talk kind and a species ID.
+    // Saved at [NA]0x204897C, which is done by the special process INIT_MAIN_TEAM_AFTER_QUIZ. This
+    // remains unchanged even if Chimecho Assembly entry 1 is modified in any way.
     VAR_PARTNER_TALK_KIND = 67,
+    // Determines which species the actor ACTOR_RANDOM_REQUEST_NPC03 will appear as. Saved at
+    // [NA]0x2065BA4.
+    // True to its name, this variable is randomly generated in the function [NA]0x205EB5C, which is
+    // called within GenerateDailyMissions. There also seems to be an exception where the game
+    // doesn't generate the species randomly (for Magnemite, maybe).
     VAR_RANDOM_REQUEST_NPC03_KIND = 68,
+    // The value of the word at [NA]0x20AFF28. Saved at [NA]0x2048990, which is done by the special
+    // process INIT_MAIN_TEAM_AFTER_QUIZ. Changing the variable doesn't seem to do much, so its true
+    // purpose is unknown.
     VAR_CONFIG_COLOR_KIND = 69,
+    // Its purpose is unknown, but it's initialized to 0 at [NA]0x204B0E4.
     VAR_ROM_VARIATION = 70,
```

Comment:
Wasn't this the one used to signal if the game is the demo version or not? (1 = Demo, 0 = Release)

Reply:
I dunno, but @Adex-8x might.

Reply:
At the very least, setting this variable to 1 (whether by a script command or editing it in RAM directly) doesn't affect the `variation` check used by the script engine, so I'm not sure on this.